src/PockyBot.NET/Services/Pegs/PegRequestValidator.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Linq;
+using GlobalX.ChatBots.Core.Messages;
+using Microsoft.Extensions.Options;
+using PockyBot.NET.Configuration;
+using PockyBot.NET.Constants;
+using PockyBot.NET.Persistence.Repositories;
+
+namespace PockyBot.NET.Services.Pegs
+{
+    internal class PegRequestValidator : IPegRequestValidator
+    {
+        private readonly PockyBotSettings _settings;
+        private readonly IConfigRepository _configRepository;
+
+        public PegRequestValidator(IOptions<PockyBotSettings> settings, IConfigRepository configRepository)
+        {
+            _settings = settings.Value;
+            _configRepository = configRepository;
+        }
+
+        public bool ValidatePegRequest(Message message, out string errorMessage)
```

Comment:
Not sure how much I like the out parameter.....
Could just throw errors for error flow control instead

Reply:
Done